### ISSUE TITLE
zsh: set zsh as the default shell, LD_PRELOAD is not set when login

### ISSUE THREAD

**Issue by moeshin** [termux/termux-packages#6756]
**Problem description**

I use `chsh -s zsh` set zsh as the default shell.
LD_PRELOAD is not set when login.

**Steps to reproduce**

I think there is a problem here: https://github.com/termux/termux-packages/blob/a44289837e2660e1ed12fb77bc0045d4156b3ce9/packages/termux-tools/login#L23

I delete `> /dev/null 2>&1`, relogin:

```
fdsan: attempted to close file descriptor 11, expected to be unowned, actually owned by FILE* 0x716c065018
Aborted
```

It is normal to execute alone `coreutils --coreutils-prog=true`  in zsh.
If the default shell is bash, no this problem.

**Additional information**
Termux: 0.112
Android: 11
zsh: 5.8
termux-exec: 1:0.9

**Comment by moeshin** [comment]
```
❯ termux-info
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirrors.tuna.tsinghua.edu.cn/termux/termux-packages-24 stable main
# science-repo (sources.list.d/science.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/science-packages-24 science stable
# game-repo (sources.list.d/game.list)
deb https://mirrors.tuna.tsinghua.edu.cn/termux/game-packages-24 games stable
Updatable packages:
All packages up to date
Android version:
11
Kernel build information:
Linux localhost 4.14.180-perf-gf2b4708 #1 SMP PREEMPT Thu Apr 29 15:02:33 CST 2021 aarch64 Android
Device manufacturer:
Xiaomi
Device model:
Redmi K20 Pro
```

**Comment by xeffyr** [comment]
`https://mirrors.tuna.tsinghua.edu.cn/termux/termux-packages-24` provides outdated zsh version.

Issue has been fixed in `5.8-6` but mirror has `5.8-5`. Please try to change mirror with `termux-change-repo`.
